### ISSUE TITLE
`bun ci` as an alias of `bun install --frozen-lockfile`

### ISSUE THREAD

**Issue by Pandapip1** [oven-sh/bun#5283]
### What is the problem this feature would solve?

NPM has a command, `npm ci`, that installs the exact packages in the lockfile for reproducible builds. Bun's command, `bun install --frozen-lockfile`, is much more verbose, is less well documented, requires adjustment of muscle memory, and makes migration slightly harder.

### What is the feature you are proposing to solve the problem?

A new subcommand, `bun ci`, which is just an alias for `bun install --frozen-lockfile`.

### What alternatives have you considered?

`bun install` could do a clean install by default.

**Comment by madwau** [comment]
I believe this already exists in Bun as an option:

```
bun install --frozen-lockfile
```

From the [docs](https://bun.sh/docs/cli/install):

> To install with reproducible dependencies, use --frozen-lockfile. If your package.json disagrees with bun.lockb, Bun will exit with an error. This is useful for production builds and CI environments.

**Comment by Pandapip1** [comment]
Could you make `bun ci` an alias of that, since I wasn't easily able to find that subcommand and it's one that would see really common use?

**Comment by nrmnqdds** [comment]
> I believe this already exists in Bun as an option:
> 
> ```
> bun install --frozen-lockfile
> ```
> 
> From the [docs](https://bun.sh/docs/cli/install):
> 
> > To install with reproducible dependencies, use --frozen-lockfile. If your package.json disagrees with bun.lockb, Bun will exit with an error. This is useful for production builds and CI environments.

im getting error
```
error: lockfile had changes, but lockfile is frozen
```
when trying to execute it in my github actions like this `  - run: bun install --frozen-lockfile && bun run build`

**Comment by kurucu** [comment]
> > I believe this already exists in Bun as an option:
> > ```
> > bun install --frozen-lockfile
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > From the [docs](https://bun.sh/docs/cli/install):
> > > To install with reproducible dependencies, use --frozen-lockfile. If your package.json disagrees with bun.lockb, Bun will exit with an error. This is useful for production builds and CI environments.
> 
> im getting error
> 
> ```
> error: lockfile had changes, but lockfile is frozen
> ```
> 
> when trying to execute it in my github actions like this ` - run: bun install --frozen-lockfile && bun run build`

That would happen with npm ci too - both fail with an error if your lockfile doesn't match package.json.

**Comment by VasiliyRusin** [comment]
Lots of CI scripts already use `npm ci`, of course bun has flag, but I'm sure that change `npm` to `bun` will be more clear for non JS developers or DevOpsers. Moreover, bun already do it for other `npm` commands.

**Comment by ryancastle** [comment]
`bun install --frozen-lockfile` fails if `bun.lockb` is missing so it's not really _compatible_ with npm.

**Comment by applemate** [comment]
make it please, it makes sense

**Comment by Confucian-e** [comment]
Any time plan for this feature?

**Comment by kelvinwop** [comment]
yeah... bun install --frozen-lockfile is long af why not just bun ci its so short and memorable

**Comment by RawToast** [comment]
In the meantime, you could add the setting to your `bunfig.toml` https://bun.sh/docs/runtime/bunfig#install-frozenlockfile

Or just add a script for the `ci` command

**Comment by raphael22** [comment]
+1
Useful to migrate preinstall hook script.

**Comment by kiranojhanp** [comment]
> yeah... bun install --frozen-lockfile is long af why not just bun ci its so short and memorable

Just write a makefile

**Comment by PratikDev** [comment]
+1
